app/src/main/java/org/fossasia/openevent/app/data/repository/AttendeeRepository.java

```diff
@@ -0,0 +1,87 @@
+package org.fossasia.openevent.app.data.repository;
+
+import android.support.annotation.NonNull;
+
+import org.fossasia.openevent.app.data.contract.IUtilModel;
+import org.fossasia.openevent.app.data.db.contract.IDatabaseRepository;
+import org.fossasia.openevent.app.data.models.Attendee;
+import org.fossasia.openevent.app.data.models.Attendee_Table;
+import org.fossasia.openevent.app.data.network.EventService;
+import org.fossasia.openevent.app.data.repository.contract.IAttendeeRepository;
+import org.fossasia.openevent.app.utils.Constants;
+
+import javax.inject.Inject;
+
+import io.reactivex.Observable;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+
+public class AttendeeRepository extends Repository implements IAttendeeRepository {
+
+    @Inject
+    public AttendeeRepository(IUtilModel utilModel, IDatabaseRepository databaseRepository, EventService eventService) {
+        super(utilModel, databaseRepository, eventService);
+    }
+
+    @NonNull
+    @Override
+    public Observable<Attendee> getAttendee(long attendeeId, boolean reload) {
```

Comment:
In future, if we need to show the attendee detail page, we'll need to change the contract, which is bad